common/constants/constants.go

```diff
@@ -278,6 +278,12 @@ var (
 	// 		:: Default = 6.4*1e8
 	COINBASE_PAYOUT_AMOUNT = uint64(6.4 * 1e8)
 
+	// Limited number of 100 fed servers in the network at 0% efficiency
+	COINBASE_PAYOUT_SERVER_LIMIT = uint64(100) * COINBASE_PAYOUT_AMOUNT
```

Comment:
The correct ceiling for the payouts have to be determined as there is almost no coming back from this once it's rolled out.